doc/whats_new/v1.0.rst

```diff
@@ -207,6 +207,14 @@ Changelog
   for non-English characters. :pr:`18959` by :user:`Zero <Zeroto521>`
   and :user:`wstates <wstates>`.
 
+:mod:`sklearn.calibration`
+............................
+
+- |Fix| :class:`calibration.CalibratedClassifierCV` now delegating validation
+  to base estimator to fix :func:`CalibratedClassifierCV.predict` and
+  :func:`CalibratedClassifierCV.predict_proba`
```

Comment:
I think it's more important to describe what bug the fix fixes, rather than how it fixes the bug. So I'd suggest something like:

> The `predict` and `predict_proba` methods of :class:`calibration.CalibratedClassifierCV can now properly be used on prefitted pipelines.